provision/testdata/base_case_snat.kube.yaml

```diff
@@ -10,7 +10,27 @@ spec:
     plural: acicontainersoperators
     singular: acicontainersoperator
   scope: Namespaced
-  version: v1alpha1
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    schema:
+      openAPIV3Schema:
+        description: acicontainersoperator owns the lifecycle of ACI objects in the cluster
+        properties:
+          spec:
+            description: AciContainersOperatorSpec defines the desired spec for ACI Objects
+            properties:
+              flavor:
+                type: string
+              config:
+                type: string
+            type: object
```

Comment:
I don't see this change. Are we sure we don't need this. As aciContainersOperator has the status field and should be included in the schema

status:
  description: AciContainersOperatorStatus defines the successful completion of AciContainersOperator
  properties:
    status:
      type: boolean
type: object

Reply:
likewise everywhere

Reply:
Posted!